lib/Runtime/Stack/Frame/index.ts

```diff
@@ -47,8 +47,8 @@ class Frame {
   public variables: Store;
 
   constructor(frameState: Options) {
-    if (frameState.hasOwnProperty('blockID')) this.nodeID = frameState.blockID;
-    if (frameState.hasOwnProperty('nodeID')) this.nodeID = frameState.nodeID;
+    if (Object.prototype.hasOwnProperty.call(frameState, 'blockID')) this.nodeID = frameState.blockID;
```

Comment:
Why make these changes?

Reply:
it was because I've had an error with the linter, so surfing on the internet people suggests this change @DecathectZero can you check this out?

Reply:
solved